plugins/providers/docker/action.rb

```diff
@@ -66,14 +66,11 @@ def self.action_provision
                 next
               end
 
-              b3.use Call, HasSSH do |env3, b4|
-                if env3[:result]
-                  b4.use Provision
-                else
-                  b4.use Message,
-                    I18n.t("docker_provider.messages.provision_no_ssh"),
-                    post: true
+              b3.use Call, HasProvisioner do |env3, b4|
+                if env3[:skip].length > 0
+                  b4.use Message, I18n.t("docker_provider.messages.not_provisioning", provisioiners: check_skipped_provisioners(env3[:skip]))
```

Comment:
Typo on `provisioners`:

```suggestion
                  b4.use Message, I18n.t("docker_provider.messages.not_provisioning", provisioners: check_skipped_provisioners(env3[:skip]))
```